### PR TITLE
Textbox

### PR DESCRIPTION
**Pull request by nishasen**
Added text box of basic style, different types of input and with validation.
The textboxes has hovering effects and focus effects.
No width has been specified for the text boxes.
The playground.html file is just for testing purpose so it can be ignored.
Added textboxes with labels inside with validation fields too.
textbox1
textbox2
textbox3

### CODE REVIEW

**Comment by anushasunil** [comment]
looks good

File: Stylesheet/Components/textbox.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    outline: transparent;
+    border: 1px solid var(--st-darkgrey);
+}
+
+.error-outlined:hover {
+    border: 1px solid var(--st-darkred);
+}
+
+.basic-textfield-filled:hover {
+    outline: transparent;
```
Comment (sadathshariff) [inline Stylesheet/Components/textbox.css:87]: There are some properties that are repeating, you can look into that and reuse it

File: Stylesheet/Components/textbox.css [hunk 2]
```diff
@@ -0,0 +1,137 @@
+/* This is textbox file */
+
+input {
+    outline: none;
```
Comment (sadathshariff) [inline Stylesheet/Components/textbox.css:4]: outline: none is not a good UX, instead you can try element: focus{ outline: none} this is what I found 
Removing outlines in CSS creates issues for people navigating the web with a keyboard.

Comment (nishasen) [reply Stylesheet/Components/textbox.css:4]: Okay then I will make it transparent

**Review by sadathshariff — APPROVED** [review]
Looks fine,good to merge

**Pull request merged**